cli/init.md

```diff
@@ -183,19 +201,217 @@ The command should be used in the following cases:
+`project` level means the configuration is managed at the project level by the CLI, each project get its own independent configuration. The following attributes are used only when the configuration is at project level<br/>
+- `useProfile`: <br/> 
+A boolean indicating weather to use a profile defined in the shared config file (`~/.aws/config`) and credentials file (`~/.aws/credentials`). <br/>
+- `profileName`: <br/> 
+The name of the profile if `useProfile` is set to true. <br/>
+- `accessKeyId`: <br/> 
+The aws access key id if `useProfile` is set to false. <br/>
+- `secretAccessKey`: <br/> 
+The aws secret access key if `useProfile` is set to false. <br/>
+- `region`: <br/> 
+The aws region if `useProfile` is set to false. <br/>
+
+### Sample script
+```bash
+#!/bin/bash
+set -e
+IFS='|'
+
+REACTCONFIG="{\
+\"SourceDir\":\"src\",\
+\"DistributionDir\":\"build\",\
+\"BuildCommand\":\"npm run-script build\",\
+\"StartCommand\":\"npm run-script start\"\
+}"
+AWSCLOUDFORMATIONCONFIG="{\
+\"configLevel\":\"project\",\
+\"useProfile\":false,\
+\"profileName\":\"default\",\
+\"accessKeyId\":\"headlessaccesskeyid\",\
+\"secretAccessKey\":\"headlesssecrectaccesskey\",\
+\"region\":\"us-east-1\"\
+}"
+AMPLIFY="{\
+\"projectName\":\"headlessProjectName\",\
+\"envName\":\"myenvname\",\
+\"defaultEditor\":\"code\"\
+}"
+FRONTEND="{\
+\"frontend\":\"javascript\",\
+\"framework\":\"react\",\
+\"config\":$REACTCONFIG\
+}"
+PROVIDERS="{\
+\"awscloudformation\":$AWSCLOUDFORMATIONCONFIG\
+}"
+
+amplify init \
+--amplify $AMPLIFY \
+--frontend $FRONTEND \
+--providers $PROVIDERS \
+--yes
+```
+
+### The `amplify configure project` parameters
+The `amplify configure project` command allows the user to change the configuration settings that were first set by `amplify init`, and it takes the same parameters as the `amplify init` command: 
+- `--amplify`
+- `--frontend`
+- `--providers`
+- `--yes`
+
+### Sample script
+```bash
+#!/bin/bash
+set -e
+IFS='|'
+
+REACTCONFIG="{\
+\"SourceDir\":\"src\",\
+\"DistributionDir\":\"build\",\
+\"BuildCommand\":\"npm run-script build\",\
+\"StartCommand\":\"npm run-script start\"\
+}"
+AWSCLOUDFORMATIONCONFIG="{\
+\"configLevel\":\"project\",\
+\"useProfile\":false,\
+\"profileName\":\"default\",\
+\"accessKeyId\":\"headlessaccesskeyid\",\
+\"secretAccessKey\":\"headlesssecrectaccesskey\",\
+\"region\":\"us-east-1\"\
+}"
+AMPLIFY="{\
+\"projectName\":\"headlessProjectName\",\
+\"defaultEditor\":\"code\"\
+}"
+FRONTEND="{\
+\"frontend\":\"javascript\",\
+\"framework\":\"react\",\
+\"config\":$REACTCONFIG\
+}"
+PROVIDERS="{\
+\"awscloudformation\":$AWSCLOUDFORMATIONCONFIG\
+}"
+
+amplify configure project \
+--amplify $AMPLIFY \
+--frontend $FRONTEND \
+--providers $PROVIDERS \
+--yes
+```
+
+### The `amplify push/publish` parameters
+The `amplify push` command takes the following parameters, because `amplify publish` internally executes `amplify push` first, it also takes the same parameters for the `push` part. 
```

Comment:
The `amplify push` command takes the following parameters, because `amplify publish` internally executes `amplify push` first, it also takes the same parameters for the `push` part 
--> 
The `amplify publish` command internally executes `amplify push` so it takes the same parameters as push command. The `amplify push` command takes the following parameters: